File: YTMSiparis/KullanıcıUyg/python/Adres.py
class Adres:
    def __init__(self, ulke=None, il=None, ilce=None, posta_kodu=None):
        self._ulke = ulke
        self._il = il
        self._ilce = ilce
        self._posta_kodu = posta_kodu

    @property
    def ulke(self):
        return self._ulke

    @ulke.setter
    def ulke(self, value):
        self._ulke = value

    @property
    def il(self):
        return self._il

    @il.setter
    def il(self, value):
        self._il = value

    @property
    def ilce(self):
        return self._ilce

    @ilce.setter
    def ilce(self, value):
        self._ilce = value

    @property
    def posta_kodu(self):
        return self._posta_kodu

    @posta_kodu.setter
    def posta_kodu(self, value):
        self._posta_kodu = value

    def __str__(self):
        return f"{self.ulke}, {self.il}, {self.ilce}, Posta Kodu: {self.posta_kodu}"

File: YTMSiparis/KullanıcıUyg/c#/SRP/SRP/Adres.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SRP
{
    public class Adres
    {
        private string il {  get; set; }
        private string ulke {  get; set; }
        private string ilce {  get; set; }
        private int postaKodu {  get; set; }
        public string Ulke
        {
            get { return ulke; }
            set { ulke = value; }
        }

        public string Il
        {
            get { return il; }
            set { il = value; }
        }

        public string Ilce
        {
            get { return ilce; }
            set { ilce = value; }
        }

        public int PostaKodu
        {
            get { return postaKodu; }
            set { postaKodu = value; }
        }

        public override string ToString()
        {
            return $"{ulke}{il}{ilce}{postaKodu}";
        }
    }
}
